browser/brave_rewards/rewards_service_factory.cc

```diff
@@ -90,18 +77,9 @@ RewardsServiceFactory::RewardsServiceFactory()
 #if BUILDFLAG(ENABLE_GREASELION)
   DependsOn(greaselion::GreaselionServiceFactory::GetInstance());
 #endif
-
-#if BUILDFLAG(BRAVE_REWARDS_ENABLED) && !defined(OS_ANDROID)
-  if (g_browser_process && g_browser_process->profile_manager())
-    g_browser_process->profile_manager()->AddObserver(this);
-#endif
 }
 
 RewardsServiceFactory::~RewardsServiceFactory() {
```

Comment:
Nit: I guess this could just be `= default` now?

Reply:
Yup good idea, fixed.